### DISCUSSION TITLE
Getting Nightly Build for Rasppios

### DISCUSSION THREAD

**Discussion by larryiweaver** [AlexandreRouma/SDRPlusPlus#765] [Q&A]
Github has always been a mystry to me. When I check for a new Nightly Build for Raspios I go to the Nightly Build section, click on "build_raspios_bullseye_armhf" and then look at the date which is Apr 21. To me that says there has not been a new nightly build for a while. Am I doing this correctly. I really like SDR++ and want to keep up with the latest and greatest.

Thank you,
Larry

**Comment by AlexandreRouma** [comment]
the latest nightly is indeed that old. I'm currently extremely busy with university so there has not been any work done on the project since then.

**Reply by larryiweaver** [reply]
Hello Alexandre,
      Thank you for the reply. I'm somewhat aware of your extremely busy schedule. I was not complaining in any way, I just wanted to know if I am checking correctly. Thank you very much for all that you have done. Software like SDR++ is well worth the wait, whatever it is.

Regards,
Larry
